boto3/docs/service.py

```diff
@@ -120,3 +128,16 @@ def _document_resources(self, section):
             ResourceDocumenter(
                 resource, self._botocore_session).document_resource(
                     section.add_new_section(resource.meta.resource_model.name))
+
+    def _get_example_file(self):
+        return os.path.realpath(
+            os.sep.join([self.EXAMPLE_PATH,
+                         self._service_name + '.rst']))
```

Comment:
FWIW, I would prefer to use os.path.join(..., ..., ...) in favor of os.sep.join([..., ..., ...]), because it is conceptually a high level API.

This is NOT a must-fix. I will still give out a "ship-it" for this PR based on its current implementation.